Common.cmake

```diff
@@ -46,6 +46,9 @@ set(CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS_RELWITHDEBINFO} -DNDEBUG")
 set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} -DNDEBUG")
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(CMAKE_INSTALL_MESSAGE LAZY) # no up-to-date messages on installation
+if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
+  unset(INSTALL_PACKAGES CACHE) # Remove after install in SubProject.cmake
```

Comment:
is there no way it can be done inside SubProject.cmake? These kind of cross-file operations variables are always hard to track and maintain...